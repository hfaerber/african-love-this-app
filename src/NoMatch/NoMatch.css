.div-invalid-route {
  width: 100%;
  border: 6px ridge #f28242;
  height: 95vh;
  padding: 5%;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
}

.h2-not-found {
  color: #f28242;
}

.h2-invalid-route {
  font-style: italic;
}
