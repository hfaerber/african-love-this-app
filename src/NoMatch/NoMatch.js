import React from 'react';
import { Link } from 'react-router-dom';
import './NoMatch.css';

const NoMatch = () => {
  return(
    <div className='div-invalid-route'>
      <h2 className='h2-invalid-route'>Wow. You really do need help with Geography.</h2>
      <h2 className='h2-not-found'>Page Not Found</h2>
      <h2 className='h2-invalid-route'>How did you even get here?</h2>
      <Link to ='/'>
        <button className='button-invalid'>Let's try again</button>
      </Link>
    </div>
  )
}

export default NoMatch;
