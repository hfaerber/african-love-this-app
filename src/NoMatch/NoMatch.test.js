
import React from 'react';
import NoMatch from './NoMatch';
import { shallow } from 'enzyme';

describe('NoMatch', () => {
  let wrapper;

  it('should match snapshot', () => {
    wrapper = shallow(<NoMatch />);
    expect(wrapper).toMatchSnapshot();
  });
});
