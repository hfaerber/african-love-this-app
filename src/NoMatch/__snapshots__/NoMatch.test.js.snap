// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch should match snapshot 1`] = `
<div
  className="div-invalid-route"
>
  <h2
    className="h2-invalid-route"
  >
    Wow. You really do need help with Geography.
  </h2>
  <h2
    className="h2-not-found"
  >
    Page Not Found
  </h2>
  <h2
    className="h2-invalid-route"
  >
    How did you even get here?
  </h2>
  <Link
    to="/"
  >
    <button
      className="button-invalid"
    >
      Let's try again
    </button>
  </Link>
</div>
`;
